.gitignore

```diff
@@ -10,3 +10,4 @@ molecule/default/tests/__pycache__
 share/
 
 super-linter.log
+.vscode
```

Comment:
I haven't though about it this way. 
Usually on public repos open to contributors, you might want to avoid everyone committing it's own file editor configs so you dont end up with a load of _dotfiles_ floating around in the project. 
I use VScode as well but I didn't need your config to compel with ansible-lint or other checks.

Reply:
> Usually on public repos open to contributors, you might want to avoid everyone committing it's own file editor configs

Yeah fair point. Happy to leave it with your change in then.